test/certs/gencerts.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -eu
+
+if [ ! -f gleam.toml ]; then
+  echo "Please run this script in the project root" >&2
+fi
+
+pushd test/certs
+if [[ -f server.crt || -f server.key || -f server.csr || -f ca.crt || -f ca.key ]]; then
+  popd
+  echo "A key already exists in test/keys. Proceeding will overwrite it!"
+  echo "Press ENTER to proceed, CTRL+C to cancel."
+  read
+  pushd test/certs
+fi
```

Comment:
since this is mostly gonna be used on CI, I suggest you detect CI here and overwrite files anyway, otherwise just fail and make the user manually remove the certs.

Most (all?) CI systems export a `CI` environment variable?
what do you think?

ps.
I was inspired to look here by the somewhat unusual `yes | …` usage :D

Reply:
you also probably don't need the pushd/popd calls, since when you execute the script it wont modify the calling shells working directory

Reply:
Fixed.